apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/first-post-published-modal/index.tsx

```diff
@@ -54,6 +53,7 @@ const FirstPostPublishedModal: React.FC = () => {
 			shouldShowFirstPostPublishedModal &&
 			! previousIsCurrentPostPublished.current &&
 			isCurrentPostPublished &&
+			launchpadScreenOption !== 'full' &&
```

Comment:
This reads that the 'first post publised modal' wont appear when the launchpad option is full. IIRC, this modal is intended for the `write` intent regardless of the launchpad setting?  [This comment](https://github.com/Automattic/jetpack/blob/f624366798f0f74d3df3cb8ddce7728c6a4b1282/projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js#L132) on the jetpack side seems to suggest this first publish modal should take precedence. cc @autumnfjeld

Reply:
That's right, , the modal is intented for the `write` flow as [showed here](https://github.com/Automattic/wp-calypso/blob/aeaabd9effbeaa45697d70f730ec40a6206cc22c/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-first-post-published-modal-controller.php#L86C46-L86C70)

Reply:
ahhh ok, so the fix should probably be on the launchpad side

Reply:
> ahhh ok, so the fix should probably be on the launchpad side

Probably, although its in Jetpack and constrained by that release cycle along with Serenity finishing a meetup. I wonder if doing something like this here might be a good temporary solution while those changes come thought.